.github/workflows/sycl-linux-build.yml

```diff
@@ -70,7 +70,7 @@ on:
       build_image:
         type: choice
         options:
-          - "ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build"
+          - 'ghcr.io/intel/llvm/sycl_ubuntu2404_nightly:latest'
```

Comment:
Should we have `ghcr.io/intel/llvm/ubuntu2404_intel_drivers:alldeps` here as well? I guess alldeps is required for building E2E tests?

Reply:
We want this to be a nightly image so that one could manually dispatch to build with clang. And nightly images have been unified into a single `:latest` tag in https://github.com/intel/llvm/pull/16680. I haven't done the same for "base" images but intent to do so shortly.